config/routes.rb

```diff
@@ -174,7 +174,8 @@
       end
     end
 
-    resources :version, only: %i[show], constraints: ->(req) { req.format == :json }, param: :version_id, as: 'version_json'
+    resources :version, controller: 'versions', only: %i[show], constraints: ->(req) { req.format == :json }, param: :version_id, as: 'version_json'
```

Comment:
I'm not sure how/if the `version_json` route helper is getting used currently. It seems leftover from some previous implementation (which may have gotten overwritten, leading to this issue).

Reply:
I see, it gives us `item_version_json_path`.